src/payloads/copy_message.rs

```diff
@@ -8,14 +8,14 @@
 // [`schema`]: https://github.com/WaffleLapkin/tg-methods-schema
 use serde::Serialize;
 
-use crate::types::{ChatId, Message, MessageEntity, ParseMode, ReplyMarkup};
+use crate::types::{ChatId, MessageEntity, MessageId, ParseMode, ReplyMarkup};
 
 impl_payload! {
     /// Use this method to copy messages of any kind. The method is analogous to the method forwardMessage, but the copied message doesn't have a link to the original message. Returns the [`MessageId`] of the sent message on success.
     ///
     /// [`MessageId`]: crate::types::MessageId
     #[derive(Debug, PartialEq, Eq, Hash, Clone, Serialize)]
-    pub CopyMessage (CopyMessageSetters) => Message {
+    pub CopyMessage (CopyMessageSetters) => MessageId {
```

Comment:
These files are auto generated, can you undo changes in this file and fix this in the schema instead?

https://github.com/WaffleLapkin/tg-methods-schema